src/vs/workbench/contrib/comments/browser/commentThreadWidget.ts

```diff
@@ -149,12 +149,9 @@ export class CommentThreadWidget<T extends IRange | ICellRange = IRange> extends
 	}
 
 	updateCommentThread(commentThread: languages.CommentThread<T>) {
-		if (this._commentThread !== commentThread) {
-			dispose(this._commentThreadDisposables);
-		}
-
 		this._commentThread = commentThread;
 		this._commentThreadDisposables = [];
+		dispose(this._commentThreadDisposables);
```

Comment:
Shouldn't this line go before 153?

Reply:
@rzhao271 good catch, moved to the wrong line 👍